System Analysis/system_analysis.py

```diff
@@ -862,41 +959,58 @@ def character_level(model_name, type):
 
     tot = len(actpred[actpred["same?"] == 0].index)
 
-    c_cons = len(cons_df.index)
-    sent = (
-        "Single Letter Consonant Error Words: \n"
-        + str(cons_df.head())
-        + "\n"
-        + str(cons_df.describe())
-        + "\n"
-    )
-    # print(sent)
-    analysis.append(sent)
-
-    c_dias = len(dias_df.index)
-    sent = (
-        "Single Letter Diacritic Error Words: \n"
-        + str(dias_df.head())
-        + "\n"
-        + str(dias_df.describe())
-        + "\n"
-    )
-    # print(sent)
-    analysis.append(sent)
-
-    c_inde = len(inde_df.index)
-    sent = (
-        "Single Letter Independent Error Words: \n"
-        + str(inde_df.head())
-        + "\n"
-        + str(inde_df.describe())
-        + "\n"
-    )
-    # print(sent)
-    analysis.append(sent)
+    try:
+        c_cons = len(cons_df.index)
+        sent = (
+            "Single Letter Consonant Error Words: \n"
+            + str(cons_df.head())
+            + "\n"
+            + str(cons_df.describe())
+            + "\n"
+        )
+        # print(sent)
+        analysis.append(sent)
+    except:
+        print(
+            "No sufficient data to perform some analysis. Skipping current, moving to next analysis."
```

Comment:
Due to insufficient data, skipping some analysis and moving to the next analysis.